### DISCUSSION TITLE
Allow badges / awards to be displayed in users profile area on mobile by default

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16206] [Feature Requests]
To see badges and awards on mobile, you have to hit 'More info about'. I feel it would be more of a flex for community members if this information about them was available immediately. The profile would stand out more especially if it included whether a community member on tmmal was a black belt or pro athlete or affiliated with the UFC.

I'd argue there is a case of removing the 'More info about' button altogether.

**Comment by andypiper** [comment]
I think this is a useful, user-centric suggestion. I’d like to see more attention to the mobile web UI in general.
